### PR TITLE
Map lang and script for name.

### LINKED ISSUE #2231: Map lang and script for name
For druid:ck080vs6570:

```
Original node:
<name type="corporate" usage="primary" lang="eng" script="Latn">
  <namePart>Oji Holdings Corporation</namePart>
  <role>
    <roleTerm type="code" authority="marcrelator" authorityURI="http://id.loc.gov/vocabulary/relators/" valueURI="http://id.loc.gov/vocabulary/relators/cre">cre</roleTerm>
    <roleTerm type="text" authority="marcrelator" authorityURI="http://id.loc.gov/vocabulary/relators/" valueURI="http://id.loc.gov/vocabulary/relators/cre">creator</roleTerm>
  </role>
</name>
Best guess roundtripped node:
<name type="corporate" usage="primary">
  <namePart>Oji Holdings Corporation</namePart>
  <role>
    <roleTerm valueURI="http://id.loc.gov/vocabulary/relators/cre" authority="marcrelator" authorityURI="http://id.loc.gov/vocabulary/relators/" type="text">creator</roleTerm>
    <roleTerm valueURI="http://id.loc.gov/vocabulary/relators/cre" authority="marcrelator" authorityURI="http://id.loc.gov/vocabulary/relators/" type="code">cre</roleTerm>
  </role>
</name>
```

### PR DESCRIPTION
**Pull request by justinlittman**
closes #2231

## Why was this change made?
It slices and dices.

## How was this change tested?
Unit, sdr-deploy

## Which documentation and/or configurations were updated?
NA

### CODE REVIEW

**Comment by justinlittman** [comment]
Before:
```
Status (n=100000):
  Success:   98865 (98.865%)
  Different: 603 (0.603%)
  To Cocina error:     0 (0.0%)
  To Fedora error:     0 (0.0%)
  MODS normalizer error:     1 (0.001%)
  Missing (no descMetadata):     531 (0.531%)
```

After:
```
Status (n=100000):
  Success:   98879 (98.879%)
  Different: 589 (0.589%)
  To Cocina error:     0 (0.0%)
  To Fedora error:     0 (0.0%)
  MODS normalizer error:     1 (0.001%)
  Missing (no descMetadata):     531 (0.531%)
```

File: app/services/cocina/from_fedora/descriptive/name_builder.rb [hunk 1]
```diff
@@ -89,6 +80,17 @@ def common_name(name_node, name)
           end.compact
         end
 
+        def common_lang_script(name_node)
```
Comment (jcoyne) [inline app/services/cocina/from_fedora/descriptive/name_builder.rb:83]: 👏

**Review by jcoyne — APPROVED** [review]

File: spec/services/cocina/mapping/descriptive/mods/name_spec.rb [hunk 2]
```diff
@@ -1260,8 +1260,6 @@
   end
 
   describe 'Transliterated name with role' do
-    xit 'to be mapped: status primary is inside parallelValue and sibling to it both'
```
Comment (ndushay) [inline spec/services/cocina/mapping/descriptive/mods/name_spec.rb:1263]: it's correct for this to be true?  to have status primary in two plces?

File: spec/services/cocina/mapping/descriptive/mods/name_spec.rb [hunk 3]
```diff
@@ -2541,126 +2579,6 @@
     end
   end
 
-  context 'with a translated name, no role' do
```
Comment (ndushay) [inline spec/services/cocina/mapping/descriptive/mods/name_spec.rb:2544]: why is this removed?

File: spec/services/cocina/mapping/descriptive/mods/name_spec.rb (hunk 2 above)
Comment (justinlittman) [reply spec/services/cocina/mapping/descriptive/mods/name_spec.rb:1263]: Yes. One indicates that of the contributors, which is primary. The other indicates of the contributor's names, which is primary.

File: spec/services/cocina/mapping/descriptive/mods/name_spec.rb (hunk 3 above)
Comment (justinlittman) [reply spec/services/cocina/mapping/descriptive/mods/name_spec.rb:2544]: It won't occur. The cocina should have the `status: primary` (i.e., be the roundtrip_cocina).
